contrib/ci/Dockerfile-arch

```diff
@@ -0,0 +1,10 @@
+FROM archlinux/base
+RUN rm /usr/share/libalpm/hooks/package-cleanup.hook
+RUN pacman -Syu --noconfirm
+RUN pacman -S --noconfirm base-devel
```

Comment:
OK, I see. Done locally, will be pushed when the other points will be addressed.